release-team/role-handbooks/branch-manager/README.md

```diff
@@ -207,6 +251,17 @@ The cherry pick script is also fairly self documenting in terms of how to invoke
 
 There has been quite a bit of recent discussion (see: [1](https://github.com/kubernetes/community/pull/2408), [2](https://github.com/kubernetes/community/pull/1980)) around improving both the cherry pick process process and its documentation.  Watch for and contribute to improvements.
 
+
+In the time between code thaw and the official release (when the patch release team takes over) consider the following when assessing the cherry-picks:
```

Comment:
On the part where _...(when the patch release team takes over)_ , it tells me that during the time between code thawing and the official release, that duration of time is taken over by the patch relase team.

Which is not the case, since the patch release team takes over after the official release. 

Perhaps, just like you've mentioned on Slack, add: _As soon as the release is out, the patch release team takes over_  somewhere on this paragraph ?

Reply:
Will rephrase